apps/web/src/activity/activity.css

```
@@ -91,6 +91,78 @@
+.inbox-source {
+  font-size: 11px;
+  color: var(--pl-color-fg-muted);
+  font-family: var(--pl-font-mono);
+}
+.inbox-dismiss {
+  margin-left: auto;
+}
+.inbox-text {
+  font-size: 13px;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)